curriculum/challenges/english/25-front-end-development/workshop-error-message-component/684d09cad4ec900e41db2e74.md

```diff
@@ -0,0 +1,57 @@
+---
+id: 684d09cad4ec900e41db2e74
+title: Step 11
+challengeType: 0
+dashedName: step-11
+---
+
+# --description--
+
+Right now the `div` element is placed in the upper left hand corner of the preview screen. But it would be nice to add some space to the top of the element. To apply `margin`, to an element using Tailwind CSS, you can use the following pattern: `m-number`. The number represents the fixed pixels to apply to all four sides.
```

Comment:
```suggestion
Right now the `div` element is placed in the upper left hand corner of the preview screen. But it would be nice to add some space to the top of the element. To apply `margin`, to an element using Tailwind CSS, you can use the following pattern: `m-number`, where `number` represents the fixed pixels to apply to all four sides.
```